enable user to enter q or Q to quit the program

xd3262nd opened the pull request.

Commit:
enable user to enter q or Q to quit the program

JHughes9802 commented:
Works, but the if statement is unnecessary. Would recommend adding .strip() to choice.upper() in case the user accidentally taps the spacebar, especially because it's impossible to see if there's space(s) after the response (would help all responses, not just for quitting).

xd3262nd commented:
@JHughes9802 I have tried without the if statement and it returned error. That's why I added the if statement. But I have added the `.strip()` to both of the choice input from the user

Commit:
added strip() method to the choice input

JHughes9802 commented:
That's interesting, the if statement on line 12 is necessary for you? I wonder if that has something to do with how Mac interprets data, because I'm able to run things just fine without it. Regardless, things run as intended.

xd3262nd commented:
Yes if I want to do `choice.strip().upper()` to the `menu.isvalid()` method. It gave me a NoneType error.

xd3262nd commented:
Okay I will just merge this branch then

claraj commented on ui.py:
```diff
@@ -10,9 +10,9 @@ def display_menu_get_choice(menu):
         choice = input('Enter choice? ')
 
         if type(choice) == str:
```
There's no way for choice to not be a string. input return strings.

xd3262nd replied:
Yes, you are right. Totally miss that one out. Thanks for pointing it out!
